### DISCUSSION TITLE
The fixed version has been provided, but it has not been officially launched yet in centos7

### DISCUSSION THREAD

**Discussion by m1ssya** [aquasecurity/trivy#5380] [Q&A] [answered]
### Description

Hello, I used trivy to scan a base image of Centos in version 7.9.2009. Scan out the vulnerability of CVE-2023-3341, and the fix version provided by trivy is 9.11.4-26.P2. el7_ 9.15, but the official version of Centos7 does not currently have this patch.

### Desired Behavior

For Centos7 images, if the official patch for Centos7 is not available, the fixed version should not be displayed in the scan results.

### Actual Behavior

The scan results provide a fixed version of CVE-2023-3341, but this fixed version cannot be obtained by yum.

### Reproduction Steps

```bash
none
```

### Target

None

### Scanner

None

### Output Format

None

### Mode

None

### Debug Output

```bash
none
```

### Operating System

macOS big sur

### Version

```bash
0.43.0
```

### Checklist

- [ ] Run `trivy image --reset`
- [ ] Read [the troubleshooting](https://aquasecurity.github.io/trivy/latest/docs/references/troubleshooting/)

**Comment by m1ssya** [comment]
![image](https://github.com/aquasecurity/trivy/assets/71381448/ff44ef52-c531-4d94-aeec-54224e895f56)

**Comment by DmitriyLewen** [comment, marked as answer]
Hello @m1ssya 
Thanks for your report!

We use RedHat advisory database for CentOS's - https://aquasecurity.github.io/trivy/v0.46/docs/scanner/vulnerability/#os-packages

[RHSA-2023:5691](https://access.redhat.com/errata/RHSA-2023:5691) for RHEL 7 says that fixed version for `bind-license` is `bind-license-9.11.4-26.P2.el7_9.15.noarch.rpm `. 
Thanks is why Trivy shows this version as fixed versions.

Rehards, Dmitriy.

**Reply by m1ssya** [reply]
Thank you very much for your reply!

**Reply by knqyf263** [reply]
Security patches in CentOS can have some delays.
